specs/MultiProfile.md

```diff
@@ -169,20 +169,37 @@ void ScenarioCookieManagement::DeleteAllCookies()
 ### Delete profile
 
 ```cpp
-HRESULT AppWindow::DeleteProfile(ICoreWebView2Controller* controller)
+HRESULT AppWindow::DeleteProfile(ICoreWebView2* webView2)
 {
-    wil::com_ptr<ICoreWebView2> coreWebView2;
-    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
-    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
-    if (webview7)
+    DCHECK(webView2);
+    wil::com_ptr<ICoreWebView2Profile> profile;
+    CHECK_FAILURE(webView2->get_Profile(&profile));
+    CHECK_FAILURE(profile2->Delete());
+}
+
+void AppWindow::RegisterProfileDeletedEventHandlers(ICoreWebView2* webView2)
+{
+    wil::com_ptr<ICoreWebView2Profile> profile;
+    CHECK_FAILURE(webView2->get_Profile(&profile));
+    if (profile)
```

Comment:
Can this be null if get_Profile succeeded?  (I don't recall if wv2s can be without a profile.  The c# sample assumes .Profile is non-null)

Reply:
if get_Profile succeeded, the profile will not be NULL, and will remove `if (profile)`.